packages/react-native/package.json

```diff
@@ -40,8 +40,12 @@
     "url": "https://github.com/knocklabs/javascript/issues"
   },
   "peerDependencies": {
-    "react": ">=16.8.0",
-    "react-native": ">=0.60.0"
+    "expo": "~50.0.14",
+    "expo-constants": "^15.4.0",
+    "expo-device": "^5.9.3",
+    "expo-notifications": "^0.27.6",
```

Comment:
I think including these as peer dependencies may be incorrect since these are only required when using the expo provider?

Reply:
@kylemcd In this case, how would you recommend declaring these dependencies?